aws-android-sdk-iot/src/main/java/com/amazonaws/mobileconnectors/iot/AWSIotMqttManager.java

```diff
@@ -1176,6 +1253,10 @@ void reconnectToSession() {
                 case CUSTOM_AUTH:
                     options.setCustomWebSocketHeaders(customWebsocketHeaders);
                     break;
+                case USERNAME_PASSWORD:
+                    options.setUserName(username);
```

Comment:
Is this duplicative of the work done around line 1059?

Reply:
This is setting options for reconnection as opposed to 1059 where it was for first connection attempt.